source/bao_hyp/user_manual/porting.rst

```diff
@@ -0,0 +1,270 @@
+Porting
+=======
+
+The Bao Hypervisor only contains architecture-specific drivers (e.g., interrupt controllers or
+IOMMUS). Therefore porting the hypervisor essentially resumes to two simple operation: creating a
+platform description folder containing a platform definition source and header files which
+indicates, among others, the number of CPUs or the memory available. The exception is an UART driver
+used fore logging, so if a driver is not available for a UART on the target platform this might
+require writing such a driver. Finally, given for some platforms provide a custom mechanism to
+define DMA device Master IDs required for configuring IOMMUs, one might require to configure
```

Comment:
At this point, it would be interesting to refer to some examples to contextualize the firmware changes